projects/components/src/stepper/stepper.component.ts

```diff
@@ -0,0 +1,130 @@
+import {
+  AfterContentInit,
+  ChangeDetectionStrategy,
+  Component,
+  ContentChildren,
+  EventEmitter,
+  Input,
+  Output,
+  QueryList
+} from '@angular/core';
+import { MatStepper } from '@angular/material/stepper';
+import { queryListAndChanges$ } from '@hypertrace/common';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { ButtonRole, ButtonStyle } from '../button/button';
+import { IconSize } from '../icon/icon-size';
+import { StepperTabComponent } from './stepper-tab.component';
+
+@Component({
+  selector: 'ht-stepper',
+  template: `
+    <div class="stepper" *ngIf="this.steps$ | async as steps">
```

Comment:
Probably you can use a LoadAsync or LetAsync here. The LoadAsync will you a spinner until the observable is resolved

Reply:
+1

Reply:
Agreed, Updated. Thanks!